build-logic/jvm/src/main/kotlin/gradlebuild.unittest-and-compile.gradle.kts

```
@@ -298,7 +298,7 @@ val Project.maxTestDistributionPartitionSecond: Long?
     get() = providers.systemProperty("testDistributionPartitionSizeInSeconds").forUseAtConfigurationTime().orNull?.toLong()
 
 val Project.maxParallelForks: Int
-    get() = (findProperty("maxParallelForks")?.toString()?.toInt() ?: 4) * (if (System.getenv("BUILD_AGENT_VARIANT") == "AX41") 2 else 1)
+    get() = findProperty("maxParallelForks")?.toString()?.toInt() ?: 4
```

```suggestion
    get() = findProperty("maxParallelForks")?.toString()?.toInt() ?: 8
```

Shouldn't the default be `8` now?

---

Well, on CI it's controlled by global `maxParallelForks`, but on other places (like local machines, it probably should not change.)

---

Right. So that looks good!